LocalWiki.php

```diff
@@ -52,7 +52,10 @@ function onBeforePageDisplay( OutputPage $out ) {
 	}
 }
 
-if ( $wgDBname === 'polandballwikisongcontestwiki' ) {
+if ( $wgDBname === 'polandballwikisongcontestwiki' ||
```

Comment:
This can be simplified to `in_array( $wgDBname, [ ... ] )` it'd probably be better that way, but not a blocker for deployment of this. If you don't want to I will merge right now though.